src/qt/transactionfilterproxy.cpp

```diff
@@ -51,8 +52,11 @@ bool TransactionFilterProxy::filterAcceptsRow(int sourceRow, const QModelIndex &
         return false;
     if(datetime < dateFrom || datetime > dateTo)
         return false;
-    if (!address.contains(searchString, Qt::CaseInsensitive) && !label.contains(searchString, Qt::CaseInsensitive))
+    if (!address.contains(searchString, Qt::CaseInsensitive)
```

Comment:
Avoid unnecessary code execution?
```cpp
if (!searchString.isEmpty()) {
    ...
}
```

Reply:
address.contains("") will immediately return true, so the other two clauses won't get executed. I don't know that we need to over-optimise here.

Reply:
Ok, not so immediately as `isEmpty` but agree with you.